lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -180,8 +179,8 @@ if Code.ensure_loaded?(Membrane.WebRTC.EndpointBin) do
     end
 
     @impl true
-    def handle_notification({:vad, val}, {:endpoint, endpoint_id}, _ctx, state) do
-      send(state.owner, {:vad_notification, val, endpoint_id})
+    def handle_notification({:vad, val}, :endpoint_bin, _ctx, state) do
+      send(state.owner, {:vad_notification, val, state.ice_name})
```

Comment:
As I mentioned in other PR, I think we should use `name` from `ctx` instead of `state.ice_name`